docs/installation.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Install and uninstall BTP Manager
+---
+
+## Install BTP Manager locally
+
+### Prerequisites
+
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- Kubernetes cluster (you can use [k3d](https://k3d.io)) 
+
+Use the following commands to run the BTP Manager controller from your host. Both `make` commands refer to [Makefile](../Makefile) in the `operator` directory.
```

Comment:
`operator` dir doesn't exist anymore

Reply:
@szwedm Is the Makefile okay?

Reply:
@IwonaLanger yes, referenced Makefile is correct.

Reply:
Thank you @szwedm